sites/friday/src/get-started/here/set-resources.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Managing resources
+sidebarTitle: Resources
+weight: 40
+description: Flexible resources on {{% vendor/name %}} are easy to use. Find all you need to know about resources allocation there.
+---
+
+To define allocated resources to your services and apps, you need to define how much CPU, memory, and disk to assign to the various containers.
+Back in your terminal, run:
+
+```bash {location="Terminal"}
+{{% vendor/cli %}} resources:set
+```
+
+This will launch an interactive prompt to walk you through setting up your application's resources:
+```bash {location="Terminal"}
+{{% vendor/cli %}} resources:set
+Resource configuration for the project app (123456azerty), environment main (type: production):
++-----------------------+---------+---------+-------------+-----------+-----------+
+| App or service        | Size    | CPU     | Memory (MB) | Disk (MB) | Instances |
++-----------------------+---------+---------+-------------+-----------+-----------+
+| app                   | not set | not set | not set     | N/A       | 1         |
+| [<additional-service>]| not set | not set | not set     | not set   | 1         |
++-----------------------+---------+---------+-------------+-----------+-----------+
+```
+The first question is what profile size you want applied to your application image. For now let's select the minimum `0.1`:
```

Comment:
should we define, or at least link, to an explanation of profiles? We're walking through the steps of altering their resources, but gloss over what a profile is, and what profile types they have available to them.